.travis.yml

```diff
@@ -1,17 +1,21 @@
 language: node_js
 dist: trusty
 sudo: required
-node_js: 6
+node_js: stable
+env:
+  - PATH=$HOME/purescript:$PATH
 install:
-  - npm install bower -g
+  - TAG=$(wget -q -O - https://github.com/purescript/purescript/releases/latest --server-response --max-redirect 0 2>&1 | sed -n -e 's/.*Location:.*tag\///p')
+  - wget -O $HOME/purescript.tar.gz https://github.com/purescript/purescript/releases/download/$TAG/linux64.tar.gz
+  - tar -xvf $HOME/purescript.tar.gz -C $HOME/
+  - chmod a+x $HOME/purescript
+  - npm install -g bower
   - npm install
-  - bower install --production
+  - bower install
 script:
   - npm run -s build
-  - bower install
-  - npm -s test
```

Comment:
Yeah, I'm not sure what to do with Travis in general here, until there is a Pulp update.